googleinterns/smart-home-insurance-discounts-app
# Integration testing 1 [P0]

Test 1 added:
Checks flow login->policy->discount->payment

1.  Start at login page
2. Find "Login with Google" button and click on it
3.  Find sidebar button and click on it. Select "Purchase Policy"
4.  Find address textboxes and fill them. Click on submit.
5.  Find radio buttons and policy names and select a policy
6.  Find "Smart Discounts" button and click on it
7.  Find "Add Devices" button and click on it
8.  Find the first a structure and select
9. TODO:  Find list of offers and confirm that only offers that can be availed are present
10.  Select an offer and click on "Go to Payment"
11. TODO:  Check if address, offers, structure and policies displayed are the ones chosen.
12.  Find "Pay" button and click on it
13.  Check if redirected to home page and Logout
14. Check if redirected to login screen (Failing currently: 3P app needs fix)

In order to run test: flutter drive --target=test_driver/app.dart

Working Video:
https://drive.google.com/file/d/1OoAr_eDOsiJE7hF0GAwxg2myiMRQGjku/view?usp=sharing
